rotkehlchen/tests/unit/test_globaldb_upgrades.py

```diff
@@ -93,3 +94,14 @@ def test_upgrade_v2_v3(globaldb):
         ).fetchone()[0]
         # should have found one asset that FLO's swapped_for is pointing to
         assert found_assets == 1
+
+        # Check that new evm tokens have been correctly upgraded in price_history. Checking BIFI
+        cursor.execute('SELECT price FROM price_history WHERE from_asset == "eip155:56/erc20:0xCa3F508B8e4Dd382eE878A314789373D80A5190A"')  # noqa: E501
```

Comment:
do you also check that BIFI history had 'BIFI' as identifier before the upgrade?

Reply:
Added the check before migrating